carta/session.py

```diff
@@ -417,6 +417,43 @@ def clear_raster_scaling_reference(self):
         """Clear the raster scaling reference."""
         self.call_action("clearRasterScalingReference")
 
+    # VIEWER MODES
+    @validate(Boolean())
+    def set_viewer_mode(self, multiple=True):
```

Comment:
The frontend implementation uses a boolean to switch in between the two viewer modes. This does not mean we should expose this boolean to users. set_viewer_mode(True) / set_viewer_mode(False) is not intuitive and explicitly express the purpose of the function. I suggest we have a keyword argument as mode with options "single" and "multiple". Under the scene we can still use the boolean for control.

Reply:
Done in ab3d8af (14f6ca5, actually). The original `if_multiple` argument is changed to `panel`.  The new `panel` argument is allowed to set `"single"` and `"multiple"`. If it's not these two inputs, a logger. warning will appear to notify the user to set the correct input.